site/content/contribute/why-contribute/_index.md

```diff
@@ -0,0 +1,99 @@
+    2. If you see a problem with any webpages, blog posts, or other content on [Mattermost.com](http://Mattermost.com), you can notify us via [the Content channel](https://community.mattermost.com/core/channels/mattermost-blog).
+    3. Share your contributor or user experience! Mention us when you promote work you do in our community and we’ll amplify the message through Mattermost social media profiles.
+    4. Want to lead a social community? We can provide advice and resources to help you in the [Community channel](https://community.mattermost.com/core/channels/community-team).
+
+## Subsection 3 — You’d like to make something more inclusive or accessible
+
+1. Accessibility is one of the most overlooked yet most important features of modern software development, and we’re eager to improve on the accessibility of the features in our open source project and its documentation.
+2. Problems with the accessibility or the inclusivity of both features in the codebase and pieces of content describing Mattermost are bugs, so we treat them as such in our development process. See subsections 1 and 2 for the description on how you can submit your contribution.
+3. When you contribute a change that incorporates an adjustment based on the principles of accessibility and inclusivity, please use this guide to back you up in the PR, ticket, or post.
+
+## Subsection 4 — You have a feature idea
+
+1. Thank you for your enthusiasm! You can act on feature ideas in a few ways:
+    1. Take a look at our [product roadmap](https://mattermost.com/roadmap/). There’s a chance we might already be building the thing you want.
+    2. [Provide input on features](https://portal.productboard.com/mattermost/33-what-matters-to-you/tabs/117-potential-future-offerings) we’re considering to let us know what matters the most to you.
+    3. [Participate in a survey](https://portal.productboard.com/mattermost/33-what-matters-to-you/tabs/115-help-us-learn-more-through-surveys) to help us better understand how to meet the needs of our users.
+    4. Discuss your idea with our community in the [Feature Proposals channel](https://community.mattermost.com/core/channels/feature-ideas).
+    5. Build an app! Mattermost has a rich framework full of tools to help you add the features you want that don’t quite work as core additions to Mattermost.
+        1. [Webhooks]({{< ref "/integrate/webhooks/" >}}) are the easiest type of app to create and enable you to integrate with the vast majority of modern web tools.
+        2. [Slash commands]({{< ref "/integrate/slash-commands/" >}}) are apps that respond to triggers sent as messages inside Mattermost.
+        3. More complicated apps can be built as standard HTTP services in any programming language using a robust API. [Get started here]({{< ref "/integrate/apps/" >}}).
```

Comment:
```suggestion
        3. More complicated Apps can be built as standard HTTP services in any programming language using a robust API. [Get started here]({{< ref "/integrate/apps/" >}}).
```
We capitalize the `A` in `Apps` when referring to the Apps framework vs. other uses of the word `app`.